### PR TITLE
feat: add InputRefSelect to components

### PR DESCRIPTION
**Pull request by connoratrug**

### CODE REVIEW

**Review by mswertz — CHANGES_REQUESTED** [review]
still doesn't work on safari, neither on preview nor when I checked out code and run locally.

**Review by MaxPostema — CHANGES_REQUESTED** [review]
- For me safari also did not work and firefox had some issues. Chrome works nice.
- Maybe use string template literal for (id) string concatenation?

File: apps/molgenis-components/src/components/forms/InputRefSelect.vue [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    filter: Object,
+    refLabel: String,
+  },
+  computed: {
+    title() {
+      return "Select " + this.table;
+    },
+  },
+  methods: {
+    flattenObject(objectToFlatten) {
```
Comment (MaxPostema) [inline apps/molgenis-components/src/components/forms/InputRefSelect.vue:93]: Lots of components use this function. It has been moved to the utils file
/components/utils.js

Comment (connoratrug) [reply apps/molgenis-components/src/components/forms/InputRefSelect.vue:84]: it used the flattenObject form utils

**Review by mswertz — APPROVED** [review]

**Pull request merged**